README.md

```diff
@@ -1,2 +1,11 @@
-# relativepath.tech
-The RelativePath Website
+# React + TypeScript + Vite
+
+Template from [here](https://github.com/tailwindtoolbox/Landing-Page/blob/master/index.html).
+
+## Usage
+
+**Assumes you have Node installed. You can get it for your machine [here](https://nodejs.org/en/download)**
+
+- Clone this repo.
+- Navigate to repo and run `npm install` to install dependencies.
+- Run `npm run dev` to start server and press the "o" key or navigate to http://localhost:5173 in your browser.
```

Comment:
This should be automated where some of it is dependencies and some of this is testing like "curl the site, expect 200".

Reply:
This has been automated with `test.sh`